video/cloud-client/analyze/analyze_test.py

```diff
@@ -24,37 +24,37 @@
 BUCKET = os.environ['CLOUD_STORAGE_BUCKET']
 LABELS_FILE_PATH = '/video/cat.mp4'
 FACES_FILE_PATH = '/video/googlework.mp4'
-SAFE_SEARCH_FILE_PATH = '/video/cat.mp4'
+EXPLiCIT_CONTENT_FILE_PATH = '/video/cat.mp4'
```

Comment:
Nit: should the `i` be lower case in `EXPLiCIT`?

Reply:
done.